app/models/spree/order_decorator.rb

```diff
@@ -34,12 +34,12 @@ def flow_order
     # accepts line item, usually called from views
     def flow_line_item_price(line_item, total = false)
       result = if flow_order
-                 id = line_item.variant.id.to_s
+                 id = line_item.variant.sku
 
                  lines = flow_order.lines || []
                  item  = lines.select { |el| el['item_number'] == id }.first
```

Comment:
While we are here, I've noticed also this:

```suggestion
                 item  = lines.find { |el| el['item_number'] == id }
```

Reply:
Ohhh great call! I ll make this change, and do a quick test locally (just in case). If everything goes ok I will merge this PR